Chat/Network.cpp

```diff
@@ -145,6 +145,31 @@ void Network::SendFile(std::string pass, std::string ip)
 	 file_send_socket_.SendFile(pass, ip);
 }
 
+std::vector<std::string> Network::GetFileList()
```

Comment:
I think, it is a little expensive to copy all vector. Maybe, there is the way to return reference?